iso15118/shared/security.py

```diff
@@ -125,8 +126,14 @@ def get_ssl_context(server_side: bool) -> Optional[SSLContext]:
          Need to figure out a way to securely store those certs and keys
          as well as read the password.
     """
-    # TODO In ISO 15118-20, we use TLS 1.3. Need to adapt that later
-    ssl_context = SSLContext(protocol=PROTOCOL_TLSv1_2)
+
+    if ENABLE_TLS_1_3:
+        ssl_context = ssl.create_default_context(
+            purpose=Purpose.CLIENT_AUTH if server_side else Purpose.SERVER_AUTH,
+            cafile=CertPath.OEM_ROOT_PEM if server_side else CertPath.V2G_ROOT_PEM,
```

Comment:
shouldnt it be:

```python
cafile=CertPath.V2G_ROOT_PEM if server_side else CertPath.OEM_ROOT_PEM,
```

Reply:
It looks like caFile points to the certificate required to verify the peer's chain ...
https://docs.python.org/3/library/ssl.html#ssl.create_default_context

Reply:
as discussed, the create_default_context already does the same settings as later done in the code; instead we can do the following:

```python
if ENABLE_TLS_1_3:
    ssl_context = SSLContext(protocol=ssl.PROTOCOL_TLS)
else:
    ssl_context = SSLContext(protocol=ssl.PROTOCOL_TLSv1_2)
```

Reply:
Sure, done.